config/application.rb

```diff
@@ -93,6 +93,7 @@ class Application < Rails::Application # rubocop:todo Style/Documentation
     config.cherrypickable_default_type = 'ABgene_0800'
     config.plate_default_type = 'ABgene_0800'
     config.plate_default_max_volume = 180
+    config.plate_default_control_wells_to_leave_free = [19, 79].freeze
```

Comment:
Is not clear what they refer these positions (d1? f2?). Could we add a comment?

Reply:
Good call, added it in the commit message, but can't harm to have it in the code itself.

Reply:
Referenced the issue as well